packages/medulas-react-components/src/components/Drawer/index.tsx

```diff
@@ -134,23 +135,26 @@ function PersistentDrawerRight({ children, items, elevation = 0 }: Props): JSX.E
         open={open}
         classes={drawerClasses}
       >
-        <div className={classes.drawerHeader}>
-          <IconButton onClick={handleDrawerClose}>
-            <ChevronRightIcon />
-          </IconButton>
+        <div id={DRAWER_HTML_ID}>
+          <div className={classes.drawerHeader}>
+            <IconButton onClick={handleDrawerClose}>
+              <ChevronRightIcon />
+            </IconButton>
+          </div>
+          <List component="nav">
+            {items.map(item => (
+              <ListItem button key={item.text}>
+                {item.icon && (
+                  <ListItemIcon>
+                    <Image src={item.icon} alt="Menu icon" />
+                  </ListItemIcon>
+                )}
+                <ListItemText primary={item.text} onClick={item.action} />
+              </ListItem>
+            ))}
+          </List>
+          {footer && <footer>{footer}</footer>}
```

Comment:
Conceptually this is not entirely correct, footer normally is used to specify a region of webapp, normally at the bottom of the view. In this case, Drawer is a component which has a vertical space, where, in our case, we place links (with an icon or without it) inside it. Seeing the screenshots I do not see the connection with a footer.

From a user perspective, I would not be happy if I use a Drawer and it has a... footer. 

Anyway do not change it, it works, and it works fine.

Reply:
Hmm, a drawer is the whole element that slides into the screen, which gave it the name:
https://en.wikipedia.org/wiki/Drawer_(furniture)#/media/File:Drawer.agr.jpg

The material design guide describes headers inside of Drawers, but not explicitely footers. So even if not the most typical thing, I don't see what is wrong with having a footer analogue to a header. Regarding the HTML `<footer>` element, I don't care much which element is used. I think it is pretty useful for element selections in tests but it could be changed easily.

Reply:
The material design guidelines say a header is allowed if for any reason the drawer's location collides with a real header in the app. In this header region you can place your logo for example.

But it does not say you can use the <header> tag. In fact, if you use them you are committing SEO penalty, and lower page rating.

Same applies to the footer, there is absolutely no reference about it, the closest reference is [this](https://www.dropbox.com/s/zleo52p992zn1ep/Screenshot%202019-07-04%2013.45.01.png), and it talks navigation bar.

I do not want you to expend time to justify your decisions, I was off, so I was not able to review this PR, just wanted to leave a comment just in case we need to do something else here, to remember it, and refactor. Just that. 

Good job on the FE code btw.